### PR TITLE
feat: User Info Page with dummy data

### PR DESCRIPTION
**Pull request by lizacullis**

### CODE REVIEW

File: services/web-app/src/app/profile/page.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      </>
+    );
+  }
+
+  return (
+    <>
+      <h1 className="text-3xl flex justify-right mt-10 mb-5 ml-10">
+        My Profile
+      </h1>
+      <div className="flex flex-col p-5 ml-10 mr-10">
```
Comment (SEBRATHEZEBRA) [inline services/web-app/src/app/profile/page.tsx:42]: Also here, can change it to `mx-10` :)

File: services/web-app/src/components/buttons/signIn.tsx [hunk 2]
```diff
@@ -2,8 +2,8 @@ import { signIn } from 'next-auth/react';
 
 export const SignInButton = () => {
 return (
-  <button className="border-2 p-[5px] text-xl hover:underline" onClick={() => { signIn("github") }}>
+  <button className="border-2 p-[5px] mr-3 ml-3 text-xl hover:underline" onClick={() => { signIn("github") }}>
```
Comment (SEBRATHEZEBRA) [inline services/web-app/src/components/buttons/signIn.tsx:5]: And here `mx-3`

File: services/web-app/src/components/tables/repoTable.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { Table } from "@radix-ui/themes";
+
+interface RepoTableProps {
+  repos: string[];
+}
+
+export const RepoTable: React.FC<RepoTableProps> = ({ repos }) => {
+  return (
+    <div className="ml-10 mr-10">
```
Comment (SEBRATHEZEBRA) [inline services/web-app/src/components/tables/repoTable.tsx:9]: Lastly here to `mx-10`

**Review by SEBRATHEZEBRA — COMMENTED** [review]
Just some tailwind changes, otherwise looks pretty good to me :)

**Comment by mattzcarey** [comment]
Can we see a screenshot of this in the PR. Helps a lot for approving FE tickets

File: services/web-app/src/app/profile/page.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    "make-money-fast-crypto",
+  ];
+
+  const { data: session, status } = useSession();
+
+  if (status === "loading") {
+    return <Loading />;
+  }
+
+  if (!session || session === null) {
```
Comment (mattzcarey) [inline services/web-app/src/app/profile/page.tsx:21]: epic!

Comment (mattzcarey) [inline services/web-app/src/app/profile/page.tsx:21]: you shouldnt need the session === null. null is a falsy value so this is included in !session

**Review by mattzcarey — APPROVED** [review]
Please fix small thing

**Comment by lizacullis** [comment]
> Can we see a screenshot of this in the PR. Helps a lot for approving FE tickets

Screenshot 2023-08-21 at 10 56 37

**Pull request merged**